packages/istanbul-lib-instrument/package.json

```diff
@@ -14,11 +14,11 @@
     "prepublish": "npm run release"
   },
   "dependencies": {
-    "babel-generator": "^6.18.0",
-    "babel-template": "^6.16.0",
-    "babel-traverse": "^6.18.0",
-    "babel-types": "^6.18.0",
-    "babylon": "^6.18.0",
+    "@babel/generator": "7.0.0-beta.49",
+    "@babel/template": "7.0.0-beta.49",
+    "@babel/traverse": "7.0.0-beta.49",
+    "@babel/types": "7.0.0-beta.49",
+    "babylon": "7.0.0-beta.47",
```

Comment:
Oh we renamed, which is why the versions don't match. It's just `@babel/parser` now.

Reply:
interesting, switching to `@babel/parser` seems to break shebang support:

```bash
  1) statement.yaml/shebang code
       covers line and one branch:
     AssertionError: Error compiling
1: #!/usr/bin/env node
2: 
3: var x = args[0] > 5 ? args[0] : "undef";
4: output = x;
5: 
Invalid or unexpected token: expected false to be truthy
```

is there an additional plugin that's required? seems to be  a regression from babylon@7.0.0-beta.47.

Reply:
Hmm interesting. I'll take a look. I did land a change for shebang handling, but I can't tell from that error if it's a bug, or just not expecting the new logic.

Reply:
interesting, it doesn't seem to be shebangs in general, this works:

```js
// #!/usr/bin/env node
var x = args[0] > 5 ? args[0] : "undef";
output = x;
```

but this doesn't work:

```js
#!/usr/bin/env node
var x = args[0] > 5 ? args[0] : "undef";
output = x;
```

Reply:
Okay, I see what's up. In Babel 6 the interpreter directive was basically silently converted into a LineComment in the AST. 

In 7.x, we changed is that now if your input file had a shebang, it actually gets passed to the code generator and generated as a shebang again. That's causing problems for your logic here: https://github.com/istanbuljs/istanbuljs/blob/ac824a46b25a815977394d96843a31657a2be77b/packages/istanbul-lib-instrument/test/util/verifier.js#L127 that means the interpreter directive is no longer at the start of the file, and thus a syntax error.

Reply:
Maybe your tests should just strip directives in that logic?